### PR TITLE
Fixes #79 - Proccessing susi response for hyperlinking urls

### LINKED ISSUE #79: Processing response for hyperlinking urls
If the response from susi contains urls, they could be replaced by a phrase like "Click Here" hyperlinked to that url.

### PR DESCRIPTION
**Pull request by uday96**
Proccesses the responses from susi and replaces any urls with "Click Here" hyperlinking that url

### CODE REVIEW

**Comment by rishiraj824** [comment]
@uday96 Can you provide a demo link?

File: src/components/MessageListItem.react.js [hunk 1]
```diff
@@ -1,17 +1,35 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
+const processString = require('react-process-string');
```
Comment (rishiraj824) [inline src/components/MessageListItem.react.js]: Also try using ```import processString from 'react-process-string';``` to import files following the ES6 syntax.

**Comment by uday96** [comment]
@rishiraj824 @isuruAb will make a pr using a different module, as pointed out react-process-string module is causing the build to fail due to incompatible syntax :+1:

**Comment by rishiraj824** [comment]
@uday96 Please squash the commits. And provide the demo link.

**Comment by uday96** [comment]
@isuruAb  @rishiraj824 Im having connectivity issues deploying it for a demo link.
http://webchatdemo.surge.sh/ is the link for the previous commit. Here the url opens in the same tab. I changed that by adding target="_blank" in the recent commit. WIll post a demo link for the recent one soon.
Also, All kinds of url links and also mail ids are hyperlinked but I couldnt replace it with a phrase like "Click Here" . WIll work on that too.

**Pull request merged**